app/views/clients/rollup/_zip_map.html.haml

```diff
@@ -1,13 +1,22 @@
 -# %img{ src: "//maps.googleapis.com/maps/api/staticmap?#{params.map(&:to_query).join('&')}", style: 'display:block;margin: 0 auto;' }
-- addresses = @client.previous_permanent_locations_for_display(current_user).select{|m| m[:lat].present?}
-- markers = addresses.map{|m| [m[:lat], m[:lon]]}.uniq
-- labels = addresses.map{|m| m[:label]}.uniq
+:ruby
+  addresses = @client.previous_permanent_locations_for_display(current_user).select{|m| m[:lat].present?}
+  markers = addresses.map{ |m| {lat_lon: [m[:lat], m[:lon]], label: m[:label]} }.uniq
+  max_lat = addresses.map{ |m| m[:lat] }.max
+  min_lat = addresses.map{ |m| m[:lat] }.min
+  max_lon = addresses.map{ |m| m[:lon] }.max
+  min_lon = addresses.map{ |m| m[:lon] }.min
+  bounds = [
+    [min_lat, min_lon],
+    [max_lat, max_lon],
+  ]
```

Comment:
```
lat = addresses.map{ |m| m[:lat] }
lon = addresses.map{ |m| m[:lon] }
bounds = [
  [lat.min, lon.min],
  [lat.max, lon.max],
]
```
To avoid redundant maps?